packages/lib-user/src/utils/getStatsDateString.js

```diff
@@ -1,6 +1,10 @@
+/*
+  Stats dates are always handled in UTC strings such as 24-09-23.
```

Comment:
Similar to other comment
```suggestion
  Stats dates are always handled in UTC strings such as 2024-09-23.
```

Reply:
Shoot I just realized I never included these suggested changes. I'll include it in https://github.com/zooniverse/front-end-monorepo/pull/6472